package/Shaders/Common/SharedData.hlsli

```diff
@@ -136,13 +137,22 @@ struct PBRSettings
 	uint pad0[2];
 };
 
+struct TransclucencySettings
+{
+	uint AlphaMode;
+	float AlphaReduction;
+	float AlphaSoftness;
+	uint pad0;
+};
+
 cbuffer FeatureData : register(b6)
 {
 	GrassLightingSettings grassLightingSettings;
 	CPMSettings extendedMaterialSettings;
 	CubemapCreatorSettings cubemapCreatorSettings;
 	TerraOccSettings terraOccSettings;
 	LightLimitFixSettings lightLimitFixSettings;
+	TransclucencySettings transclucencySettings;
```

Comment:
The setting buffers seems not working if I put it after wetnessEffectsSettings.
I suspect there is a misalign between the HLSL buffer def and the cpp version of it. But couldn't pin it out so far.